src/features/currentTodo.ts

```diff
@@ -25,7 +25,11 @@ type Action = SetTodoAction | RemoveTodoAction;
 
 const currentTodoReducer = (state: State = null, action: Action): State => {
   switch (action.type) {
-    // Implement all actions here
+    case 'currentTodo/SET':
```

Comment:
Pay attention
![2024-03-21_16h39_03](https://github.com/mate-academy/react_redux-list-of-todos/assets/61904995/3ebb2d41-1884-4958-aa8d-bd8a057e5bdb)